diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -209,15 +237,26 @@ function StudentQuiz(runtime, element, initData) {
       //he will be resumed to where he left.
 
       resizeContentContainer();
-      var completedStep = parseInt($(completedStepSelector, element).val());
-
-      //log event for xblock started
-      common.publishEvent({
+      var eventData = {
         event_type: 'xblock.diagnostic_feedback.quiz.started',
         quiz_type: initData.quiz_type,
-        quiz_title: initData.quiz_title,
-        current_question:  completedStep + 1
-      });
+        quiz_title: initData.quiz_title
+      };
+
+      var totalQuestions = $('.question-container').length;
+      var completedStep = parseInt($(completedStepSelector, element).val()) ;
+      if (completedStep == totalQuestions){
```

Comment:
@jareerahsan Most likely a typo -- always use `===` to compare values. While you're at it, please also add a space between `)` and `{` at the end of the line (and check/adjust the rest of the JS code to make sure the formatting is consistent).